tasks/generate.py

```diff
@@ -16,6 +16,18 @@
 
 PROJECT_ROOT = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
 
+VERSION_CHECK = '''
+PY_VERSION = sys.version_info[:2]
+
+if PY_VERSION in [(2, 6), (3, 2), (3, 3)]:
```

Comment:
```suggestion
if PY_VERSION in [(2, 6), (3, 2), (3, 3), (3, 4)]:
```

Reply:
Yep, also line 37 of `get-pip.py`?

https://github.com/pypa/get-pip/pull/25/files#diff-f9d50f9e474eb60e0675070eeff17925R37